lib/internal/process.js

```diff
@@ -117,6 +118,35 @@ function setupMemoryUsage() {
   };
 }
 
+function setupResourceUsage() {
```

Comment:
This function doesn't accept any parameters. Why don't we move the array creation part also to C++?